app/controllers/sample_types_controller.rb

```diff
@@ -2,15 +2,17 @@ class SampleTypesController < ApplicationController
   respond_to :html, :json
   include Seek::UploadHandling::DataUpload
   include Seek::IndexPager
+  include Seek::AssetsCommon
 
   before_action :samples_enabled?
-  before_action :find_sample_type, only: [:show, :edit, :update, :destroy, :template_details, :batch_upload]
+  before_action :find_sample_type, only: %i[show edit update destroy template_details batch_upload]
```

Comment:
because of the order actions are run, this will likely be overridden by find_and_authorize_requested_item so it possibly no longer needed.

But needs checking because the logic is slightly different wrt isa_json_compliant sample types

Reply:
I double checked it and it doesn't get skipped. In fact, it makes some tests fail if I remove this method.